package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "scripts": {
     "dev": "gatsby develop",
-    "build": "gatsby build",
+    "build": "NODE_OPTIONS=--max-old-space-size=8192 gatsby build",
     "build:local": "./node_modules/.bin/cross-env LOCAL_IMAGES=true gatsby build",
```

Comment:
@jbukuts FYI I added `NODE_OPTIONS` to the gatsby build because my build was failing due to a javascript heap memory.